packages/a11y-testing/src/facades/ComponentTestFacade.tsx

```diff
@@ -1,7 +1,28 @@
 import * as React from 'react';
-import * as ReactDOM from 'react-dom';
 import { Props, PropValue, TestFacade } from '../types';
+import { emptyTheme, ThemePrepared } from '@fluentui/styles';
+import { Renderer, noopRenderer } from '@fluentui/react-northstar-styles-renderer';
 import { mount, ReactWrapper } from 'enzyme';
+import { Telemetry, Unstable_FluentContextProvider, ProviderContextPrepared } from '@fluentui/react-bindings';
+
+const EmptyThemeProvider: React.FunctionComponent<{
```

Comment:
This is copied from 
fluentui\packages\fluentui\react-northstar\test\utils\withProvider

Not sure if we should not extract "withProvider" to package and consume it then here and in the other unit tests as well? 

Or is it ok to have it here in this way?

Reply:
I'd vote to export and reuse here :)

Reply:
let's move ComponentTestFacade to react-bindings as it now contains a lot of v0 specific code.

Reply:
Is it required? Our components should not throw when are used without `Provider`

Reply:
For example Popup.tsx had issue in following function (figured out while debugging):
`updateTriggerFocusableRef`, or any other function, where `context.target`  is required.

Before I had `context.target`  undefined. 

When I user "withProvider" then there is wrapping component `target: document`, what allows popup works. 

@layershifter

Reply:
> let's move ComponentTestFacade to react-bindings as it now contains a lot of v0 specific code.

moved :) into: packages\fluentui\react-northstar\test\utils

Reply:
Our component should render without a `Provider`. I suggest to fix a usage in `Popup` itself:

```diff
  const updateTriggerFocusableRef = () => {
-    const activeDocument: HTMLDocument = context.target;
-    const activeElement = activeDocument.activeElement;
+    const activeElement = context.target?.activeElement;

+ if (activeElement) {
    triggerFocusableRef.current =
      triggerRef.current && elementContains(triggerRef.current, activeElement as HTMLElement)
        ? (activeElement as HTMLElement)
        : triggerRef.current;
+  }
  };
```

- typecast to `HTMLDocument` can be removed as our context now typed properly
- optional chaining in `context.target?.activeElement` is required as definitely `context.target` can be undefined:

https://github.com/microsoft/fluentui/blob/8827b3790eba246d42f34a9b16793afb9cd519ef/packages/fluentui/react-bindings/src/context.ts#L33-L34

Reply:
After this fix and next fixes pushed by @layershifter the popup works without wrapper/provider. 
Therefore returning back the 'ComponentTestFacade.tsx' into the a11y-testing package.